docs/evaluate/development-production-features/low-latency.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: low-latency
+title: Low latency - Temporal feature
+description: Discover how Temporal's design and features ensure low latency for your applications. 
+sidebar_label: Low latency
+keywords:
+  - low latency
+  - temporal performance
+  - temporal cloud
+  - self-hosted temporal
+  - workflow latency
+  - temporal benchmarks
+  - temporal sdk metrics
+  - optimized workflow performance
+  - latency-sensitive applications
+  - temporal cloud advantages
+tags:
+  - temporal-latency
+  - temporal-cloud
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Temporal provides features that significantly reduce latency, making your applications faster, more performant, and more efficient.
+In the world of modern applications, low latency is crucial for ensuring minimal delay in Workflow Executions.
+
+Temporal Cloud includes architectural advancements like a custom persistence layer, which reduce latency effectively.
+Benchmarks have shown that application-side latency is lower when using Temporal Cloud. 
```

Comment:
As compared to what? Using Temporal On-Prem? Or like, code in general?

Reply:
This is compared to self-hosted. This is what I'm using as reference: https://temporal.io/blog/benchmarking-latency-temporal-cloud-vs-self-hosted-temporal